helper/resource/testing_test.go

```diff
@@ -156,67 +158,19 @@ func TestComposeTestCheckFunc(t *testing.T) {
 
 // mockT implements TestT for testing
 type mockT struct {
-	ErrorCalled    bool
-	ErrorArgs      []interface{}
-	FatalCalled    bool
-	FatalArgs      []interface{}
-	ParallelCalled bool
-	SkipCalled     bool
-	SkipArgs       []interface{}
-
-	f bool
-}
-
-func (t *mockT) Error(args ...interface{}) {
-	t.ErrorCalled = true
-	t.ErrorArgs = args
-	t.f = true
-}
-
-func (t *mockT) FailNow() {
-	t.f = true
-
-	panic("mockT.FailNow")
-}
-
-func (t *mockT) Fatal(args ...interface{}) {
-	t.FatalCalled = true
-	t.FatalArgs = args
-	t.f = true
+	testinginterface.RuntimeT
```

Comment:
Ah I see now the data captured in the mock is removed. Honestly the 2 unit tests that use the mock are somewhat superfluous. I think I would prefer expose mitchell's lib on the SDK surface and just delete those 2 tests and `mockT` altogether.

Reply:
Considering its a `_test` package I just went for minimal change here. Its not exposed externally really and we did have testing to check if parallel was invoked, so I needed a wrapper to capture that.